tests/business.test.ts

```diff
@@ -39,7 +39,8 @@ import { Marqeta } from '../src';
   console.log('getting list of Businesses...')
   const list = await client.business.list()
 
-  if (list?.body?.isMore) {
+  if (list.body?.isMore) {
+    console.log(`[IS MORE]: ${JSON.stringify(list.body.isMore)}`)
```

Comment:
This could look better... but it's up to you...

Reply:
nah, that was debug statement.  Removed.